infrastructure/services/task.go

```diff
@@ -355,9 +338,21 @@ func (l TaskServiceLocker) Create(owner string, taskQueue string, timeout int32,
 	return l.next.Create(owner, taskQueue, timeout, retry, payload, notBefore)
 }
 func (l TaskServiceLocker) Select(taskID string) error {
+	ctx := context.Background()
+	lock, err := l.acquire(ctx, taskID)
+	if err != nil {
+		return err
+	}
+	defer func() { lock.Release(ctx) }()
```

Comment:
Je ne pense pas que la closure soit nécessaire ici (potentiellement y'en a d'autres à changer dans le code ajouté).

```suggestion
	defer lock.Release(ctx)
```